rollup.config.js

```diff
@@ -9,7 +9,7 @@ export default [
       file: 'dist/index.js',
       format: 'cjs',
     },
-    external: ['path', 'child_process'],
+    external: ['path', 'child_process', 'os', 'fs', 'events', 'util'],
```

Comment:
Where are `events ` and `util` imported?

```suggestion
    external: ['path', 'child_process', 'os', 'fs'],
```

Reply:
Ah ok - yeah thats cool, its just hard for maintainers to know where they come from - perhaps a comment?

Reply:
Pushed a commit to add the comment and (while I was there) avoid some duplication in the same config file.